<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="cwMain">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="hlFrame">
      <item>
       <widget class="QLabel" name="lblFrame">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblGray">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlSlider">
      <item>
       <widget class="QSlider" name="slFrame">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbxFrameCurrent"/>
      </item>
      <item>
       <widget class="QLabel" name="lblFrames">
        <property name="text">
         <string>/ 30000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnCapture">
        <property name="text">
         <string>Capture Characters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblRemaining">
        <property name="text">
         <string>Remaining: 30</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project...</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load Project...</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionIdentify_Segments">
   <property name="text">
    <string>Identify Segments</string>
   </property>
  </action>
  <action name="actionOCR_All">
   <property name="text">
    <string>OCR All</string>
   </property>
  </action>
  <action name="actionTranslate_All">
   <property name="text">
    <string>Translate All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
